Add MTE-900 [v115] Regenerate Test-Fixture DBs

jjSDET opened the pull request:
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-900)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/FXIOS-14655)

### Description
The database strategy for storing browser data has changed, which necessitates the regeneration of test data and their dbs.

The following dbs have been regenerated, which all use the new `places.db` storage instead of the previous `browser.db` monolith.

- browserAutocomplete
- browserYoutubeTwitterMozillaExample
- exampleURLHistoryBookmark
- testBookmarksDatabase1000
- testHistoryDatabase4000
- testDatabaseFixture

A script was also created to create test dbs with seeded data for bookmarks and history for any number of records in order to support current and future performance testing.

### Pull requests checks where applicable
- [x] Fill in the three TODOs above (tickets number and description of your work)
- [x] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [ ] Unit tests written and passing
- [x] Documentation / comments for complex code and public methods

isabelrios commented:
@jjSDET great work here! thanks!I have not gone through the review of the script but I a few small comments meantime..
- We are missing the change so that we can actually use these dbs within the tests: https://github.com/mozilla-mobile/firefox-ios/blob/46fe494169d48355001b96f8f0af5be54a84f9a7/Client/Application/UITestAppDelegate.swift#L53
- Can we fix the warnings shown by danger, those related to print statements?
Thanks!

jjSDET commented:
> @jjSDET great work here! thanks!I have not gone through the review of the script but I a few small comments meantime..
> 
>     * We are missing the change so that we can actually use these dbs within the tests: https://github.com/mozilla-mobile/firefox-ios/blob/46fe494169d48355001b96f8f0af5be54a84f9a7/Client/Application/UITestAppDelegate.swift#L53
> 
>     * Can we fix the warnings shown by danger, those related to print statements?
>       Thanks!

just added logging and updated the `browser.db` to `places.db` in the `TestAppDelegate`. let me know if i missed anything else.

isabelrios reviewed (approved):
LGTM! this is great! having the dbs back that allows us to re-enable the tests and having the script to be able to generate custom dbs as we need them to try things out!!! thanks!!
